src/rpcgovernance.cpp

```diff
@@ -217,13 +218,17 @@ UniValue gobject(const UniValue& params, bool fHelp)
         std::string strVoteAction = params[2].get_str();
         std::string strVoteOutcome = params[3].get_str();
         
-        int nVoteSignal = CGovernanceVoting::ConvertVoteSignal(strVoteAction);
-        if(nVoteSignal == VOTE_SIGNAL_NONE || nVoteSignal == -1)
+        vote_signal_enum_t eVoteSignal = CGovernanceVoting::ConvertVoteSignal(strVoteAction);
+        if(eVoteSignal == VOTE_SIGNAL_NONE) {
             throw JSONRPCError(RPC_INVALID_PARAMETER, "Invalid vote signal. Please use one of the following: 'yes', 'no' or 'abstain'");
+        }
 
-        int nVoteOutcome = CGovernanceVoting::ConvertVoteOutcome(strVoteOutcome);
-        if(nVoteOutcome == VOTE_OUTCOME_NONE || nVoteOutcome == -1)
-            throw JSONRPCError(RPC_INVALID_PARAMETER, "Invalid vote outcome. Please using one of the following: (funding|valid|delete|clear_registers|endorsed|release_bounty1|release_bounty2|release_bounty3) OR `custom sentinel code` "); 
+        vote_outcome_enum_t eVoteOutcome = CGovernanceVoting::ConvertVoteOutcome(strVoteOutcome);
+        if(eVoteOutcome == VOTE_OUTCOME_NONE) {
+            throw JSONRPCError(RPC_INVALID_PARAMETER, 
+                               "Invalid vote outcome. Please using one of the following: "
+                               "(funding|valid|delete|clear_registers|endorsed|release_bounty1|release_bounty2|release_bounty3) OR `custom sentinel code` "); 
```

Comment:
Am I missing smth or are we listing here some strings we can't currently convert correctly? 

Also given the list "funding|valid|...." shouldn't this message say "signal", not "outcome"? Same question for the line 311. And lines 223 and 304 should probably say "outcome", not "signal". And related lines should probably be swapped (in pairs) to match `if`s.